lib/daru/dataframe.rb

```diff
@@ -1408,19 +1408,15 @@ def only_numerics opts={}
       Daru::DataFrame.new(arry, clone: cln, order: order, index: @index)
     end
 
-    # Generate a summary of this DataFrame with ReportBuilder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
-    end
-
-    def report_building(b) # :nodoc: #
-      b.section(name: @name) do |g|
-        g.text "Number of rows: #{nrows}"
-        @vectors.each do |v|
-          g.text "Element:[#{v}]"
-          g.parse_element(self[v])
-        end
-      end
+    # Generate a summary of this DataFrame
```

Comment:
While updating the code, please consider improving the documentation also. We use [YARD](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md) as our documentation tool. I am not saying you should write 10 paragraphs of code, but at least adding `@return` tag, and probably an example of this method's output, will be useful.